### PR TITLE
Create description of service section

### PR DESCRIPTION
**Pull request by CBinh710**
https://github.com/users/CBinh710/projects/2?pane=issue&itemId=72043674
image

### CODE REVIEW

File: practice-one/src/components/service.html [hunk 1]
```diff
@@ -0,0 +1,7 @@
+<section class="service-container">
+  <div><img src="./assets/images/service.png" alt="Service's illustration"></div>
+  <div class="service-description-container">
+    <p class="service-header">We are <span style="color: var(--color-primary);">more</span> than <span style="color: var(--color-accent);">multiple</span> service</p>
```
Comment (hungdo-agilityio) [inline practice-one/src/components/service.html]: I already pointed out in previous PR, do not use inline style

File: practice-one/src/components/service.html [hunk 2]
```diff
@@ -0,0 +1,7 @@
+<section class="service-container">
+  <div><img src="./assets/images/service.png" alt="Service's illustration"></div>
```
Comment (hungdo-agilityio) [inline practice-one/src/components/service.html]: this `div` is useless

Comment (hungdo-agilityio) [reply practice-one/src/components/service.html]: This `div` should act as like frame for image, and the image will scale to that frame, and also that way will prevent layout shift if the image cannot load

File: practice-one/src/components/service.html (hunk 1 above)
Comment (CBinh710) [reply practice-one/src/components/service.html]: thanks, i fixed it

File: practice-one/src/components/service.html (hunk 2 above)
Comment (CBinh710) [reply practice-one/src/components/service.html]: I added and div container with the same size of image

**Review by hungdo-agilityio — APPROVED** [review]

**Pull request merged**